### ISSUE TITLE
[IE9] Error thrown when opening image dialog

### ISSUE THREAD

**Issue by Reinmar** [WebSpellChecker/ckeditor-plugin-scayt#92]
1. Open e.g. http://cdn.ckeditor.com/4.4.6/full-all/samples/replacebyclass.html
1. Enable SCAYT.
1. Double click the image to open dialog.
1. Close the dialog.
1. Double click the image again.
1. Error is thrown.

```
SCRIPT5022: DOMException: INDEX_SIZE_ERR 
replacebyclass.html, line 1 character 1
```

I was able to reproduce it on IE9, but not on IE10.

**Comment by WebSpellChecker** [comment]
Thank you for reporting the issue. The fix is already implemented in SCAYT Core. It will be available in upcoming release WebSpellChecker.net version 4.7

**Comment by WebSpellChecker** [comment]
We're glad to inform you that the WebSpellChecker.net application has been released with version 4.7. The fix in the SCAYT plug-in is available in the master branch already.
